src/App.jsx

```diff
@@ -1,12 +1,14 @@
 import { useState } from "react";
 import { Routes, Route } from "react-router-dom";
 import Home from "./Components/Home/Home";
+import Article from "./Components/Article/Article";
 
 function App() {
   return (
     <>
```

Comment:
Really, since these are all already wrapped in the routes block, you don't need to also wrap it in a react fragment.